feature-detects/css/aspectratio.js

```diff
@@ -0,0 +1,38 @@
+/*!
+{
+  "name": "aspectration css property",
```

Comment:
typo :-)

Reply:
Thank you for addressing the issue is resolved 👍